packages/transport-dom/direct.ts

```diff
@@ -33,7 +38,10 @@ const directGetPort =
           error: errorToJson(ConnectError.from(error), jsonOptions),
         }),
       );
-      servicePort.postMessage(transportResponse);
+      servicePort.postMessage(
+        transportResponse,
+        isTransportStream(transportResponse) ? [transportResponse.stream] : [],
```

Comment:
This was a fix that @turbocrime identified. We'd never used the second argument to `postMessage` before, which is an array of things to be "transferred" from another stream. This became needed because I called from one service (`ViewService`) to another (`StakingService`).